### DISCUSSION TITLE
inventory management and credit notes

### DISCUSSION THREAD

**Discussion by santiparoli** [crater-invoice-inc/crater#1053] [Q&A]
When will the version that includes inventory management and credit notes be released?

**Comment by laraaj** [comment]
We also need this function, anyone interested to develop it, we can donate for that or finance it.

**Reply by z0dijak** [reply]
@laraaj we are interested to develop it. If you're still in need of that function we can connect. Reach out to me via Mail: manuel@apnd.at
